operator/cmd/mesh/upgrade.go

```diff
@@ -167,22 +165,6 @@ func upgrade(rootArgs *rootArgs, args *upgradeArgs, l clog.Logger) (err error) {
 	}
 	l.LogAndPrintf("Upgrade version check passed: %v -> %v.\n", currentVersion, targetVersion)
 
-	// Read the overridden IOP from args.inFilenames
-	overrideIOPYaml := ""
-	if args.inFilenames != nil {
```

Comment:
i don't think we can delete this. upgrade uses the user config to not bother diffing values that are changed by the user (and which therefore are not affected by change in defaults).